css/css-contain/container-queries/auto-scrollbars.html

```diff
@@ -37,15 +37,15 @@
                                "Tests do not work with overlay scrollbars");
   });
 
+  let initialScrollerWidth = scroller.clientWidth;
   test(() => {
-    assert_less_than(scroller.clientWidth, 100, "Expects a vertical scrollbar");
     assert_equals(getComputedStyle(inner).height, "50px",
```

Comment:
If there's no scrollbar to reduce the width of #container, how can the container query still match (and set the height to 50px)?

Reply:
Oh wait, I got it. The scrollbar is there initially, which makes the container query match, which makes #inner shorter, which removes the need for the scrollbar. Although this means that now the container query no longer matches, the implementation somehow guards against circular dependencies. I guess this is acceptable behavior.